test/test_view_ops.py

```diff
@@ -908,35 +908,12 @@ def run_test(device, op):
             op = partial(fn, source=0, destination=1)
             run_test(device, op)
 
-    # Testing that the generated view_copy kernel and its derivative are implemented correctly
-    def test_view_copy(self, device):
```

Comment:
This seems to be a correctness test to verify that output of `view_copy` matches `view`. Without specifying `ref` for an OpInfo, we don't get correctness test.

So we should probably add a ref to the OpInfo as `ref=lambda x: torch.view(x).clone()` or keep this test? Wdyt?

Reply:
added `ref=lambda x, newshape: np.reshape(x, newshape).copy(),`

Reply:
This test is testing three things:
- correctness against a ref
- autograd support
- view relationships are not preserved with view_copy (but they should be with view).

The new OpInfo test tests the first two items but not the third item, right? If so we should leave the tests for the third item intact

Reply:
You are right. AFAIK, 3rd isn't tested (as it is specific to *_copy ops). 

I think it would nice to have such automated test for all `*_copy` using OpInfo. Wdyt?

cc: @bdhirsh

Reply:
More automated testing is nice, but @zou3519 is correct that this test cannot be removed in this PR

Reply:
I pulled out the test changes to this draft. I will work on them there https://github.com/pytorch/pytorch/pull/89025